### DISCUSSION TITLE
Activation of Sandbxie-support certificate silently.

### DISCUSSION THREAD

**Discussion by metaad** [sandboxie-plus/Sandboxie#3655] [Q&A Feedback]
When application is launched for the first time, Below prompts come
![image](https://github.com/sandboxie-plus/Sandboxie/assets/106653425/5ac761fa-0591-4da5-b729-ed461d50a9cc)
![image](https://github.com/sandboxie-plus/Sandboxie/assets/106653425/343fe834-0e2a-46da-89ac-1dae33bdb912)

Is there any parameter to activate this silently? Is the certificate same across all the machines for one tenant or it is specific to logged in user. If it can be activated on multiple machines, then please provide us the silent parameter to achieve it.

Also is there a way to suppress auto-update(as shown below) through setting that is stored in file/registry. If yes, kindly provide us the same.
![image](https://github.com/sandboxie-plus/Sandboxie/assets/106653425/4f07e76a-e05b-401d-8c67-63fb0635b0f7)

**Comment by bastik-1001** [comment]
My guess is, that this can't be done right now, but it sounds like a feature request. Such a feature would be very useful for corporations to deploy Sandboxie-Plus on their machines.

**Reply by metaad** [reply]
I tested new version 1.13.4 released, above feature is not implemented. Kindly let us know if there is any way by which license can be activated silently. Also is the certificate same across all the machines for one tenant or it is specific to logged in user?
